src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -131,7 +132,9 @@ export default class BottomNavigationButton extends Component {
         {...other}
         onClick={this.handleChange}
       >
-        {icon}
+        <div className={iconClassName}>
```

Comment:
We can remove that extra DOM node and the associated logic change with that diff:
```diff
    icon: {
      display: 'block',
+     margin: 'auto',
    },
```